src/lib.rs

```diff
@@ -800,6 +679,7 @@ mod tests {
     use arrayvec::ArrayString;
 
     use super::*;
+    use crate::primitives::iter::{ByteIterExt, Fe32IterExt};
```

Comment:
In e03f93231e66c703c44bdbfd1218f6d19029a697:

I believe `Fe32IterExt` is not actually used.

Reply:
Clippy should warn us then, I'll investigate.

Reply:
There is a warning.